packages/components/psammead-episode-list/src/Link.jsx

```diff
@@ -1,8 +1,27 @@
 import styled from '@emotion/styled';
-import { C_METAL, C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
+import {
+  C_METAL,
+  C_POSTBOX,
+  C_WHITE,
+  C_STONE,
+} from '@bbc/psammead-styles/colours';
 
 const Link = styled.a`
-  display: block;
+  :before {
+    position: absolute;
+    inset: 0;
```

Comment:
what does this do?

Reply:
Was on Catherine's HTML - seems to be a shorthand way of setting `top`, `bottom`, `left` and `right` - https://developer.mozilla.org/en-US/docs/Web/CSS/inset

Didn't work on all browsers though, so I had to set `top`, `bottom`, `left` and `right` anyway - will remove the `inset`